core/src/main/resources/jenkins/model/Messages_fr.properties

```diff
@@ -42,6 +42,12 @@ Hudson.ReadPermission.Description=\
   &mdash; retirez ce droit \u00e0 l''utilisateur anonymous, puis \
   ajoutez le pseudo-utilisateur "authenticated" et accordez-lui le droit en lecture.
 
+Mailer.Address.Not.Configured=Adresse pas encore configur\u00e9ea <nobody@nowhere>
```

Comment:
Not sure, but that's why I hate encoded strings when it can be avoided. Shouldn't it be `configur\u00e9e` since `\u00e9` stands for é and then the final e? Hence the a is to be removed?